packages/react-native-codegen/src/parsers/__tests__/parsers-commons-test.js

```diff
@@ -97,14 +114,16 @@ describe('assertGenericTypeAnnotationHasExactlyOneTypeParameter', () => {
         typeAnnotation,
         'Flow',
       ),
-    ).toThrow(IncorrectlyParameterizedGenericParserError);
+    ).toThrowErrorMatchingInlineSnapshot(
+      `"Module testModuleName: Generic 'typeAnnotationName' must have type parameters."`,
+    );
   });
 
-  it("throws an error if typeAnnotation.typeParameters.type doesn't have the correct value depending on language", () => {
-    const moduleName = 'testModuleName';
+  it('throws an error if typeAnnotation.typeParameters.type is not TypeParameterInstantiation when language is Flow', () => {
     const flowTypeAnnotation = {
       typeParameters: {
-        type: 'TypeParameterInstantiation',
+        type: 'wrongType',
```

Comment:
This test case was ok because an Error was thrown even though the type had the correct value.
It was the fact that params was missing that caused an Error to be thrown.